test_elasticsearch/test_server/test_rest_api_spec.py

```diff
@@ -365,6 +365,9 @@ def run_match(self, action):
             ):
                 expected = re.compile(expected[1:-1], re.VERBOSE | re.MULTILINE)
                 assert expected.search(value), f"{value!r} does not match {expected!r}"
+            elif isinstance(value, list) and isinstance(expected, list):
+                assert len(value) == len(expected)
```

Comment:
Let's add a comment after this `assert` like `"Length between {value!r} and {expected!r} wasn't equal"`?